.github/workflows/_reusable_run_tests.yml

```diff
@@ -179,540 +78,19 @@ jobs:
-
-      - uses: kronenthaler/analyze-xcoderesults-action@0.1.9
-        if: ${{ inputs.wire-ios-utilities || inputs.all }}
-        with:
-          results: xcodebuild-wire-ios-utilities.xcresult
-          warningAnnotations: true
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Test WireUtilities Project
-        if: ${{ inputs.wire-ios-utilities || inputs.all }}
-        run: |
-          echo "Testing WireUtilities Project..."
-          xcodebuild test -retry-tests-on-failure -workspace wire-ios-mono.xcworkspace -scheme WireUtilities -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee -a xcodebuild-wire-ios-utilities.log | bundle exec xcpretty --report junit --output build/reports/WireUtilitiesProject.junit
-          exit ${PIPESTATUS[0]}
-
-      # WireCryptobox
-
-      - name: Build WireCryptobox
-        if: ${{ inputs.wire-ios-cryptobox || inputs.all }}
-        run: |
-          echo "WireCryptobox has changes"
-          echo "Building WireCryptobox..."
-          xcodebuild build-for-testing -workspace wire-ios-mono.xcworkspace -scheme WireCryptobox -resultBundlePath xcodebuild-wire-ios-cryptobox.xcresult -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee xcodebuild-wire-ios-cryptobox.log | bundle exec xcpretty
-
-      - uses: kronenthaler/analyze-xcoderesults-action@0.1.9
-        if: ${{ inputs.wire-ios-cryptobox || inputs.all }}
-        with:
-          results: xcodebuild-wire-ios-cryptobox.xcresult
-          warningAnnotations: true
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Test WireCryptobox
-        if: ${{ inputs.wire-ios-cryptobox || inputs.all }}
-        run: |
-          echo "Testing WireCryptobox..."
-          xcodebuild test -retry-tests-on-failure -workspace wire-ios-mono.xcworkspace -scheme WireCryptobox -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee -a xcodebuild-wire-ios-cryptobox.log | bundle exec xcpretty --report junit --output build/reports/WireCryptobox.junit
-          exit ${PIPESTATUS[0]}
-
-      # WireTransport
-
-      - name: Build WireTransport
-        if: ${{ inputs.wire-ios-transport || inputs.all }}
-        run: |
-          echo "WireTransport has changes"
-          echo "Building WireTransport..."
-          xcodebuild build-for-testing -workspace wire-ios-mono.xcworkspace -scheme WireTransport -resultBundlePath xcodebuild-wire-ios-transport.xcresult -destination 'platform=iOS Simulator,OS=${{ env.IOS_VERSION }},name=${{ env.IPHONE_MODEL }}' | tee xcodebuild-wire-ios-transport.log | bundle exec xcpretty
-
-      - uses: kronenthaler/analyze-xcoderesults-action@0.1.9
-        if: ${{ inputs.wire-ios-transport || inputs.all }}
-        with:
-          results: xcodebuild-wire-ios-transport.xcresult
-          warningAnnotations: true
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
check how to the analysis after all the tests are runned or see if doable in the run_tests of fastlane